tests/components/landslides/test_landslide_probability.py

```diff
@@ -6,10 +6,184 @@
+    spacing = (10e0, 10e0)
+    coordinates = (0.0, 0.0)
+    grid = RasterModelGrid(shape, spacing, coordinates)
+    gridnum = grid.number_of_nodes
+
+    assert gridnum == dimensions[0][0] * dimensions[0][1]
+
+    return grid, gridnum
+
+
+def calc_unit_default_value():
+
+    unit_default_value = {}
+    scatter = np.random.randint(
+        1, 10
+    )  # smallest min is the largest scatter, minus the mode cohesion
+    unit_default_value["topographic__slope"] = (0.1, 0.8)
+    unit_default_value["topographic__specific_contributing_area"] = (10, 100)
+    unit_default_value["soil__transmissivity"] = (0.1, 10)
+    unit_default_value["soil__mode_total_cohesion"] = (30, 900)
+    mode = unit_default_value["soil__mode_total_cohesion"]
+    # code used to run the model uses a random scatter <cohesion_scatter = np.random.randint(1,10)>
+    scatter = np.random.randint(
+        1, 10
+    )  # smallest min is the largest scatter, minus the mode cohesion
+    unit_default_value["soil__minimum_total_cohesion"] = (
+        (mode[0] - scatter),
+        mode[1] - scatter,
+    )
+    # large max is the largest scatter, plus the mode cohesion
+    unit_default_value["soil__maximum_total_cohesion"] = (
+        (mode[0] + scatter),
+        mode[1] + scatter,
+    )
+    unit_default_value["soil__internal_friction_angle"] = (26, 37)
+    unit_default_value["soil__thickness"] = (
+        0.1,
+        1,
+    )  # functionally setting to uniform uniform 1 meter default; coded for the option of a range of values
+    unit_default_value["soil__density"] = (
+        1999,
+        2001,
+    )  # functionally setting to uniform 2000
+    # Note on Ksat default range calculation
+    # reverse order: (see above) lowest Transmissivity value are in upper right of synthethic grid,
+    # not reversed: (see above) highest soil thickness in upper right of synthethic grid,
+    # caution creating Ksat since parameterization of T and hs will have inverse relationship on Ksat
+    # Kmin  = Tmin/hs_max   -slow- think of a deep thick clay layer
+    kmin = (
+        unit_default_value["soil__transmissivity"][0]
+        / unit_default_value["soil__thickness"][1]
+    )
+    # Kmax = Tmax/hs_min    -fast- think of a sieve or a parking lot
+    kmax = (
+        unit_default_value["soil__transmissivity"][1]
+        / unit_default_value["soil__thickness"][0]
+    )
+    unit_default_value["soil__saturated_hydraulic_conductivity"] = (kmin, kmax)
+
+    # Output value range used in tests will change as a function of input default values
+    unit_default_value["landslide__probability_of_failure"] = (0, 1)
+    unit_default_value["soil__mean_relative_wetness"] = (0, 1)
+    unit_default_value["soil__mean_watertable_depth"] = (0, 1000)
+    unit_default_value["soil__mean_recharge"] = (0, 1000)
+    unit_default_value["soil__probability_of_saturation"] = (0, 1)
+    return unit_default_value
+
+
+def get_default_nodevalues():
+    """Print out the input range for
```

Comment:
Its not clear to me that some of these get/print functions belong here. I'm not sure where they belong, though. @mcflugen ?